src/Meditation.AttachProcessService/IProcessSnapshot.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Meditation.AttachProcessService.Models;
+using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Collections.Generic;
+
+namespace Meditation.AttachProcessService
+{
+    public interface IProcessSnapshot : IDisposable
+    {
+        public ProcessId ProcessId { get; }
+
+        public IEnumerable<ModuleInfo> GetModules();
```

Comment:
[ToFix]
```EnumerateModules``` if it can be lazy.
```IReadOnly / Immutable instead of IEnumerable``` if not.

Reply:
It is lazy - renamed to `EnumurateModules`